### PR TITLE
[COREV] Add more tests for ALU and load/store instructions

### PR DESCRIPTION
**Pull request by serkm**
These tests make sure that the instructions are not available without the `xcorevalu`/`xcorevmem` features and that they are also available with the `xcorev` feature.

### CODE REVIEW

File: llvm/test/MC/RISCV/corev/mem/invalid.s [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# RUN: not llvm-mc -triple=riscv32 %s 2>&1 \
+# RUN:        | FileCheck %s --check-prefix=CHECK-ERROR
+
+# Load instructions
+
+cv.lb t0, 0(t1!)
+# CHECK-ERROR: expected ')'
```
Comment (flip1995) [inline llvm/test/MC/RISCV/corev/mem/invalid.s:7]: Ah that's interesting. The instruction format error has precedence over the feature error. This is fine IMO, but its good to have  regression test for this.

**Review by flip1995 — APPROVED** [review]
LGTM

**Comment by flip1995** [comment]
This just adds more tests. Those tests pass for me locally. @jeremybennett good to merge :+1:

**Comment by jeremybennett** [comment]
Thanks for all the hard work @SerkanMuhcu

Good to merge.

**Pull request merged**